src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -183,6 +195,13 @@ public BoundType GetBoundAttributeType(AttributeData attribute)
 			{
 				var isDbPropertyAttributeArgs = attribute.NamedArguments.Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase)).ToList();    //case insensitive check
 
+				
+				var isDBFieldInBaseAcumaticaAttribute = TypesContainsIsDBField.Any(t =>
+					t != null && IsAttributeDerivedFromClass(attribute.AttributeClass, t));
+
+				if (isDbPropertyAttributeArgs.Count == 0 && isDBFieldInBaseAcumaticaAttribute) // IsDBField property defined in base class 
+					return BoundType.DbBound;
+
 				if (isDbPropertyAttributeArgs.Count != 1)  //rare case when there are multiple different "IsDBField" considered
```

Comment:
This check can be put before the calculation of `isDBFieldInBaseAcumaticaAttribute` because you do not need to know `isDBFieldInBaseAcumaticaAttribute` to make this check.
This is also true for the next check:
```csharp
if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
					return BoundType.Unknown;  //if there is null or values of type other than bool then we don't know if attribute is bound
```